### DISCUSSION TITLE
mk2 et VE

### DISCUSSION THREAD

**Discussion by ApiFelix** [FredM67/PVRouter-3-phase#88] [Q&A]
Salut Fred
Mon install mk2 tri fonctionne à a merveillle!
Si bien qu'étant en train d'acheter un Kangoo Ze je pense a le charger avec mes panneaux

Il a une petite batterie de 35kW
Et une charge a 3.2k avec une greenbox

As tu une solution pour lui envoyer le surplus solaire ?

Merci!

**Comment by FredM67** [comment]
Hello,
Ravi que tu sois content de ton routeur mk2 !
Pour ta borne de recharge, tu as quoi exactement ? Apparemment, il y a plusieurs modèle chez Wallgreenbox.

**Comment by ApiFelix** [comment]
Pour le moment je n'ai rien
Meme pas la voiture....
Le kangoo accepte des charges a 
1.8 et 2.3 kW sur prise standard 
3 et 3.6 sur greenbox type legrand
Et 7k sur.... autre chose 

Je pars "a priori" sur une prise standard voire une greenbox 
A moins qu'une wallbox 7k mapportebplus de service 
Merci

**Reply by ApiFelix** [reply]
Je veux dire prise greenup

**Reply by FredM67** [reply]
Alors dans tous les cas, si tu veux optimiser l'affaire, il te faudra une borne de recharge triphasée qui dispose de 3 pinces de mesures.
De cette facon, tu peux optimiser l'ensemble.
Les bornes de recharge "intelligentes" fonctionnent par palier (on ne peut pas faire autrement pour charger des batteries). Ce système peut alors être mixer avec le routeur. A voir ce que tu veux prioriser, comment tu souhaites que tout cela fonctionne, ....

**Comment by ApiFelix** [comment]
En attendant ton avis je pensais faire ce que je fais avec la clim... en mieux...
 Aujourd'hui jai programmé les clim pour démarrer a 10h30
Du coups les cumulus sont chargés  du lever jusqu'à 10h30 puis demarrage des clims et ce qui reste va aux cumulus
Tout est ok comme ca (sauf si un nuage se pointe ...)

J'imaginais donner priorité a la prise VE avec une programmation dans le genre
 Si injection < 1800 alors => triac cumulus
Si >1800 activation du relai prise VE
Et donc tout ce qui est au dessus part au cumulus 

Jouable ?

**Reply by ApiFelix** [reply]
Et bien sur si ca repasse en dessous des 1800 coupure relai prise Ve

**Reply by FredM67** [reply]
En supposant que ta borne dispose de 3 pinces, on peut mixer intelligemment le routeur et la borne.
Dans les pinces de la borne, tu fais passer "à l'envers" les câbles qui sortent des triacs du routeur (en respectant les phases bien sûr).
Je prends ton exemple de 1800W de seuil pour déclenchement de borne.

1. ton surplus augmente de 0 à 1800W =>le routeur route l'intégralité vers le/les chauffe-eaux.
2. si les câbles en sortie de triac ne passaient pas à l'envers dans les pinces de ta borne, elle verra zéro surplus, puisque consommé par le routeur. Comme les câbles passent à l'envers, ta borne va voir exactement ce que tu routes, donc de 0 à 1800W.
3. Quand on arrive à 1800W de routage (donc 1800W de surplus brut avant routage), ta borne enclenche le 1er palier (elle voit 1800W).
4. Le routeur n'a donc plus de surplus puisqu'il est consommé par la borne.
5. Ta production continue d'augmenter, le surplus visible par le routeur repart donc de 0 à n W.... il route alors ce seul surplus dans ton chauffe-eau...
6. Et cela continue jusqu'au 2ème palier de ta borne et ainsi de suite

T'en penses quoi ?

**Reply by ApiFelix** [reply]
C'est une bonne option mais qui m'oblige  a acheter une box a 500€ mini
Faire la même chose sur le mk2 n'est pas possible?

**Reply by ApiFelix** [reply]
Du coups avec une simple prise

**Reply by ApiFelix** [reply]
Ou aubpire une greenup a 100€

**Reply by FredM67** [reply]
> C'est une bonne option mais qui m'oblige a acheter une box a 500€ mini Faire la même chose sur le mk2 n'est pas possible?

comme tu es en tri, il faut mesurer le surplus en tri, il n'y a pas le choix.
En ce qui concerne le routeur, en bidouillant le prog, on pourrait plus ou moins estimer grossièrement le taux de routage, je dis bien estimer grossièrement, et donc envoyer un signal à différents seuils et/ou piloter un/des relais....
Mais ca sera vraiment du très approximatif.

**Reply by ApiFelix** [reply]
Ok tantbpis... je vais regarder pour une box 
Merci !!

**Reply by ApiFelix** [reply]
Mais estimer en tri ce n'est pas ce que dait déjà le routeur poir envoyer au cumulus ?

**Reply by ApiFelix** [reply]
Ou alors est ce parce que le  routeur envoie au cumulus les watts dispos, ne coupe pas le cumulus , mesure ce qui est encore dispo et les envoie donc ne connait jamais le total dispo ?

**Reply by FredM67** [reply]
> Mais estimer en tri ce n'est pas ce que dait déjà le routeur poir envoyer au cumulus ?

non... le routeur calcule de facon continue le bilan au point d'entrée. Si le résultat est négatif, on est en injection, il commute donc un triac pour un cycle. Tant qu'on reste dans le négatif, on commute plus...
Quand on passe en positif, on diminue la commutation... et ainsi de suite.
C'est empirique.. on cherche l'équilibre.....
Si tu montes sur une poutre, ton oreille interne cherche l'équilibre... si tu penches d'un cóté, tu compenses en déplacant ton centre de gravité à l'opposé.... ton cerveau ne calcule jamais la masse qui est en trop ou la masse qu'il faut déplacer pour retrouver l'équilibre... c'est pareil avec le routeur :D

**Reply by FredM67** [reply]
> Ou alors est ce parce que le routeur envoie au cumulus les watts dispos, ne coupe pas le cumulus , mesure ce qui est encore dispo et les envoie donc ne connait jamais le total dispo ?

c'est tout à fait cela.... on envoie ou coupe pour rester à l'équilibre... mais on ne sait pas ce qu'on envoie ou coupe.

**Reply by ApiFelix** [reply]
😁👍👍👍
Je vais déjà essayer avec mon shelly 3em en inversant le sens des pinces
Merci

Et encore merci à a quentin et toi pour le boulot 
Ma femme me prend pour un génie...
Shuuut !!

**Reply by FredM67** [reply]
> 😁👍👍👍 Je vais déjà essayer avec mon shelly 3em en inversant le sens des pinces Merci
> 
> Et encore merci à a quentin et toi pour le boulot Ma femme me prend pour un génie... Shuuut !!

De rien ;-) on ne lui dira rien, lol

**Comment by ApiFelix** [comment]
salut Fred
j'ai finalement acheté un autre Shelly 3em que j'ai branché sur chacune des sorties du routeur
du coups je connais les puissances envoyées aux CE
mais en voulant contrôler tout ça je suis tombé sur un os :

il faut mettre les 2 fils (fils A/R routeur CE ACI) dans la pince pour avoir une valeur cohérente
pourquoi ? je m'attendais à avoir 0 avec cette config .

si tu peux jeter un oeil là : https://forum-photovoltaique.fr/viewtopic.php?p=783783#p783783
merci
a+
